CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
I'm instantly confused, I had to open https://keepachangelog.com/en/1.0.0/ just to understand the changelog, one benefit of individual files is that we can open new ones without clobbering old files and having to move stuff into an `## [Unreleased]` section.

If anything I would name this `not yet released` but again that fine.

Reply:
> I had to open https://keepachangelog.com/en/1.0.0/ just to understand the changelog

This is a feature not a bug? Following a standard format is not only easier on humans but enables automation like https://github.com/pl-strflt/changelog-driven-release